manager/integration/tests/common.py

```diff
@@ -2194,8 +2194,7 @@ def crash_replica_processes(client, api, volname, replicas=None,
 
     for r in replicas:
         assert r.instanceManagerName != ""
-        kill_command = "kill `ps aux | grep '" + r['dataPath'] +\
-                       "' | grep -v grep | awk '{print $2}'`"
+        kill_command = "kill `pgrep -f " + r['dataPath'] +" | grep -v grep`"
```

Comment:
Do we still need `grep -v grep`? since you already use pgrep.

Reply:
Yes, we don't need it.
Removed, thanks.